lmdeploy/lite/apis/calibrate.py

```diff
@@ -118,7 +119,11 @@ def calibrate(model: str,
               calib_samples: int = 128,
               calib_seqlen: int = 2048,
               work_dir: str = './work_dir',
-              device: str = 'cuda') -> None:
+              device: str = 'cuda',
+              w_bits: int = 4,
+              w_group_size: int = 128,
+              search_scale: bool = False,
```

Comment:
In what cases we are supposed to suggest users enabling `search_scale`?
Should we set the default value True?

Reply:
When smooth quant model accuracy is not satisfying, we recommend it. Or, when a user does not care about time consuming of `search_scale`.